code/controllers/subsystem/time_track.dm

```diff
@@ -0,0 +1,46 @@
+SUBSYSTEM_DEF(time_track)
+	name = "Time Tracking"
+	wait = 100
+	init_order = INIT_ORDER_TIME_TRACK
+	runlevels = RUNLEVEL_LOBBY | RUNLEVELS_DEFAULT
+	flags = SS_NO_INIT
```

Comment:
Any reason youre adding an init order when it has `SS_NO_INIT`?

Reply:
The original had one, I forgot to remove the init order